src/main/java/rx/internal/operators/OperatorScan.java

```diff
@@ -104,6 +104,7 @@ public void onNext(T currentValue) {
                         this.value = accumulator.call(this.value, currentValue);
                     } catch (Throwable e) {
                         child.onError(OnErrorThrowable.addValueAsLastCause(e, currentValue));
+                        return;
```

Comment:
Good catch. Could you also add `Exceptions.throwIfFatal(e);` above `child.onError` like `OperatorMap`?